cookbook/vectordb/cassandraDb.py

```diff
@@ -0,0 +1,36 @@
+from phi.agent import Agent
+from phi.knowledge.pdf import PDFUrlKnowledgeBase
+from phi.vectordb.cassandra.cassandra import CassandraDb
+import os
+
+try:
+    from cassandra.cluster import Cluster  # type: ignore
+except (ImportError, ModuleNotFoundError):
+    raise ImportError(
+        "Could not import cassandra-driver python package. " "Please install it with `pip install cassandra-driver`."
```

Comment:
`raise ImportError("Could not import cassio python package. Please install it with pip install cassio.")`